packages/qwik/src/core/use/use-task.ts

```diff
@@ -763,7 +763,8 @@ export const runComputed = (
       const stack = new Error(
         'useComputed$: Async functions in computed tasks are deprecated and will stop working in v2. Use useTask$ or useResource$ instead.'
       ).stack;
-      logOnceWarn(stack);
+      const lessScaryStack = stack ? stack.replace(/^Error:\s*/, '') : '';
```

Comment:
if there's no stack it should just return I suppose?

Reply:
yeah I guess :)

Reply:
although I guess we should just log the warning without the stack... I'll add it